src/static/index.ts

```diff
@@ -59,7 +59,7 @@ const vehicleIds: {
  * All valid job types. This should always match up to the job types in vehicle.json.
  */
 export type JobType = 'isrSearch' | 'payloadDrop' | 'ugvRescue' | 'uuvRescue'
-| 'quickScan' | 'detailedSearch' | 'guide';
+| 'quickScan' | 'detailedSearch' | 'guide' | 'geofence';
```

Comment:
I thought geofencing is supposed to be part of a start message, not an actual vehicle jobType?